src/unix/stdpaths.cpp

```diff
@@ -365,4 +365,16 @@ wxStandardPaths::MakeConfigFileName(const wxString& basename,
     return fn.GetFullName();
 }
 
+wxString wxStandardPaths::GetSharedLibrariesDir() const
+{
+    char c;
+    wxFileName::GetPathSeparator().GetAsChar( &c );
+    // While debugging libraries usually goes to "/usr/local/lib"
+    // while in production the usual place is "/usr/lib"
+#ifdef DEBUG
```

Comment:
`DEBUG` is not used nor defined anywhere.

Reply:
@vadz,
It looks like there is o definition I can use on Linux/Mac with configure build.
Neither DEBUG, not DEBUG_INFO or wxUSE_DEBUG{_INFO} are defined

Any suggestions?